Resources/Locale/ru-RU/store/uplink-catalog.ftl

```diff
@@ -302,4 +302,4 @@ uplink-barber-scissors-desc = Хороший инструмент для тог
 uplink-backpack-syndicate-name = Рюкзак Синдиката
 uplink-backpack-syndicate-desc = Лёгкий взрывоустойчивый рюкзак для хранения различных предательских штучек.
 uplink-combat-bakery-name = Набор боевой выпечки
-uplink-combat-bakery-desc = Набор подпольного печёного оружия. Содержит багет, который опытный мим может использовать в качестве меча, и пару метательных круассанов. Когда дело сделано, съешьте улики.
+uplink-combat-bakery-desc = Набор подпольного печёного оружия. Содержит меч-багет, пару метательных круассанов и микроволновую плату Синдиката для изготовления новых. Когда дело будет сделано, съешьте все улики.
```

Comment:
```suggestion
uplink-combat-bakery-desc = Набор подпольного печёного оружия. Содержит меч-багет, пару метательных круассанов и микроволновую плату Donk Co. для изготовления новых. Когда дело будет сделано, съешьте все улики.
```

Reply:
`uplink-combat-bakery-desc = A kit of clandestine baked weapons. Contains a baguette sword, a pair of throwing croissants, and a syndicate microwave board for making more. Once the job is done, eat the evidence.`